docs/docs/guides/repository.mdx

```diff
@@ -257,16 +257,16 @@ Unlike `Repository`, Infrahub does not automatically update `Read-only Repositor
 
 ## Troubleshooting repository
 
-Various issues could affect repositories. The [Repository Status](/topics/repository#repository-statuses) gives you futher details about the root cause.
+Various issues could affect repositories. The [Repository Status](/topics/repository#repository-statuses) gives you further details about the root cause.
 
-Also, directly from the UI you can access low level operations's output:
+Also, directly from the UI you can access low level operations output:
```

Comment:
Also, directly from the UI you can access the output of low level repository operations